.github/workflows/pr-all-services.yml

```diff
@@ -32,10 +32,6 @@ jobs:
         run: yarn install --prefer-offline --frozen-lockfile
 
       # This block should eventually become use lando/actions-hyperdrive@v2
-      - name: Verify Docker dependencies
```

Comment:
Would be curious why this needs to get removed.

Reply:
Github upgrades the version of Docker in their runners more frequently these days, but not all runners update at the same time. At the time of testing some were 23 and some 24 so failing tests.